test/utils.go

```diff
@@ -95,3 +97,12 @@ func IsTLSExist() bool {
 	}
 	return true
 }
+
+func inspectFilter(name, filter string) (string, error) {
+	format := fmt.Sprintf("{{%s}}", filter)
+	result := command.PouchRun("inspect", "-f", format, name)
+	if result.Error != nil || result.ExitCode != 0 {
+		return "", fmt.Errorf("failed to inspect %s: %s", name, result.Combined())
```

Comment:
could you update the error message to insert more info about filter? @ZYecho 
like `fmt.Errorf("failed to inspect %s via filter %s: %s", name, filter, result.Combined())`

Reply:
done.